From dae225ddaa00b81c8bf35717d3588680cf59f411 Mon Sep 17 00:00:00 2001
From: Sean Cross <xobs@kosagi.com>
Date: Tue, 22 Apr 2014 13:30:08 +0800
Subject: [PATCH 22/39] deb-pkg: Add support for KBUILD_DTB

Enable packaging of dtb files alongsde kernel.

If KBUILD_DTB is specified, then copy that dtb file to the resulting
kernel package.

Signed-off-by: Sean Cross <xobs@kosagi.com>

Conflicts:
	scripts/package/builddeb
---
 scripts/package/builddeb | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index 21918f3cea04..81ebd282a4bf 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -171,13 +171,6 @@ else
 fi
 cp "$($MAKE -s image_name)" "$tmpdir/$installed_image_path"
 
-if grep -q "^CONFIG_OF=y" $KCONFIG_CONFIG ; then
-	# Only some architectures with OF support have this target
-	if grep -q dtbs_install "${srctree}/arch/$SRCARCH/Makefile"; then
-		$MAKE KBUILD_SRC= INSTALL_DTBS_PATH="$tmpdir/usr/lib/$packagename" dtbs_install
-	fi
-fi
-
 if [ ! -z $KBUILD_DTB ]; then
 	if [ -e $KBUILD_DTB ]; then
 		cp $KBUILD_DTB "$tmpdir/$installed_dtb_path"
-- 
2.14.1

